config/deploy.rb

```diff
@@ -12,7 +12,6 @@
 
 # Default value for :linked_files is []
 set :linked_files, fetch(:linked_files, []).push(
-  'config/secrets.yml',
```

Comment:
not needed, value comes from vault and is set in the environment via puppet, e.g. https://github.com/sul-dlss/puppet/blob/production/hieradata/node/sul-pub-prod.stanford.edu.yaml#L67